src/reducers/conversationsReducer.js

```diff
@@ -0,0 +1,17 @@
+import { fromJS } from 'immutable';
+
+import { GET_CONVERSATION_SUCCESS } from '../actions/actionTypes';
+
+const initialState = fromJS([]);
+
+const topics = (state = initialState, { type, conversations }) => {
```

Comment:
I think you forgot to rename the reducer here after pasting 😂

Reply:
Right, fixed it.